http/src/main/resources/reference.conf

```diff
@@ -249,7 +249,7 @@ gke {
     # to be unique within a cluster.
     releaseNameSuffix = "gxy-rls"
     chartName = "galaxy/galaxykubeman"
-    chartVersion = "0.7.1"
+    chartVersion = "0.7.0"
```

Comment:
we're downgrading version?

Reply:
I think 0.7.1 was just a temp version in a separate repo that Alex made.. 0.7.0 is the version in the "real" repo.